smartsim/entity/ensemble.py

```diff
@@ -32,27 +32,18 @@
 from tabulate import tabulate
 
 from .._core._install.builder import Device
```

Comment:
This file is about to be completely removed correct?

Reply:
That's the plan! I'll delete this file and rename `_new_ensemeble.py` -> `ensemble.py` when I get rid of the old impl!

Reply:
Old impl is gone!